extensions/wikia/VideoHandlers/VideoFileUploader.class.php

```diff
@@ -135,6 +136,20 @@ public function upload( &$oTitle ) {
 			// if video already exists make sure that we are in fact changing something
 			// before generating upload (for now this only works for edits)
 			$article = Article::newFromID( $oTitle->getArticleID() );
+			//In case Article is null
+			//Log more info and throw an exception
+			if ( !( $article instanceof Article ) ) {
+				$articleId = $oTitle->getArticleID();
+				$exception = new VideoUploadFailedException( 'Video upload failed');
+				WikiaLogger::instance()->error('Video upload: title exists but article is null', [
```

Comment:
May be, but it's not what the code says :)
`if ( !( $article instanceof Article ) ) { ... }`